entries/accordion.xml

```diff
@@ -86,6 +86,21 @@
 				</desc>
 			</type>
 		</option>
+		<option name="classes" type="Object" default='Object' example-value='{ "ui-accordion-header": null }'>
```

Comment:
Well, for the getter/setter example, not for the init example, right? Even with something other then `null` we'd still overwrite the other properties when using this, right?

```js
.accordion( "option", "classes", { "ui-accordion-header": "whatever" } );
```